app/routes/_layout+/tags.$tagId_.edit.tsx

```diff
@@ -27,6 +28,7 @@ import { zodFieldIsRequired } from "~/utils/zod";
 export const UpdateTagFormSchema = z.object({
   name: z.string().min(3, "Name is required"),
   description: z.string(),
+  color: z.string().regex(/^#/).min(7),
```

Comment:
This will accept invalid values. Can you please update the regex:
```
color: z.string().regex(/^#[0-9a-fA-F]{6}$/),
```
I think this should be better.